### DISCUSSION TITLE
Support non-numeric alert data in Unified Alerting

### DISCUSSION THREAD

**Discussion by JeanneKahn** [grafana/grafana#53366] [Alerting]
As per https://github.com/grafana/grafana/issues/46429 I'd like to request supporting non-numeric alert data in Unified Alerting.

This was something that the older versions of Grafana before Unified Alerting supported; if not the usage of the data, at least the usage of a data source that produced both numeric and non-numeric data simultaneously. 

The change to Unified Alerting means that the data sources I currently use cannot be utilised in Grafana (Cloudwatch Logs), and thus I cannot upgrade my version of Grafana until this is possible.

**Comment by JeanneKahn** [comment]
Late, but this is also referred to here: https://community.grafana.com/t/no-longer-able-to-create-alert-on-cloudwatch-raw-log-output-in-grafana-8/66928

**Comment by natellium** [comment]
Hey there! We want to let you all know that we are moving to Github Issues for feature requests. You can read more about the move [here](https://github.com/grafana/grafana/discussions/73424).

I will be closing this discussion, but if you’re still interested in this request being implemented please don’t hesitate to open a [feature request](https://github.com/grafana/grafana/issues/new?assignees=&labels=&projects=&template=1-feature_requests.md). Sorry for the inconvenience, and thanks for your contributions!

cc @grafana/alerting-squad
